src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -73,6 +74,7 @@ internal sealed class TaggerProvider : IViewTaggerProvider
             [Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider,
             ISonarLanguageRecognizer languageRecognizer,
             IAnalysisRequester analysisRequester,
+            IIssueToIssueMarkerConverter converter,
```

Comment:
Future suggestion: I think it's worth having a MEF-able TextBufferIssueTrackerFactory, instead of receiving dependencies just to pass them through. Same for the AccumulatingIssueConsumer.